app/core/src/main/java/com/fsck/k9/widget/list/MessageListWidgetProvider.java

```diff
@@ -38,10 +38,19 @@ public void onUpdate(Context context, AppWidgetManager appWidgetManager, int[] a
         }
     }
 
+    //----------------------------------change LB 29.06.2018-------------------------------------
+    private PendingIntent enterpostActionPendingIntent(Context context) {
+        Intent intent = new Intent(context, MessageList.class);
```

Comment:
Unfortunately, this is not the correct Intent to open the Unified Inbox. See e.g. here:
https://github.com/k9mail/k-9/blob/fc6c47159b8cfd79dd30c21b70e9a7eb5f8654df/app/ui/src/main/java/com/fsck/k9/activity/Accounts.java#L638-L640